ui.content/src/main/content/jcr_root/conf/aem-bootcamp/settings/wcm/template-types/page/thumbnail.png.dir/.content.xml

```diff
@@ -1,7 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file is not part of the scope of the ticket

Reply:
removed